auto_nag/user_activity.py

```diff
@@ -130,10 +130,13 @@ def check_users(
 
         return user_statuses
 
-    def _get_status_from_bz_user(self, user: dict) -> UserStatus:
+    def get_status_from_bz_user(self, user: dict) -> UserStatus:
```

Comment:
Nit: no need to change name as part of this pr

Reply:
As a general rule, I try to have a test for any change in the `UserActivity` class. I changed the name here to make the method public (instead of protected) to be accessed from the test class. Yes, python does not enforce that, but it is still a smell.